packages/nimble-components/src/table/components/cell/index.ts

```diff
@@ -1,6 +1,12 @@
-import { observable } from '@microsoft/fast-element';
+import {
```

Comment:
It kinda seems like these components are geting enough behavior to have unit tests of their own separate from the table, ie to test rendering occurs when assigned a template and test behavior updating properties / observing changes. See no reason they can't be tested without the table fixture.

Reply:
Agree. I'll create an issue for us to track the creation of these tests after this PR goes in. My intent is to work on these tests immediately.